hq/app/aws/support/TrustedAdvisor.scala

```diff
@@ -54,12 +54,15 @@ object TrustedAdvisor {
     handleAWSErrs(awsToScala(client.describeTrustedAdvisorCheckResultAsync)(request))
   }
 
-  def parseTrustedAdvisorCheckResult[A <: TrustedAdvisorCheckDetails](parseDetails: TrustedAdvisorResourceDetail => A)(result: DescribeTrustedAdvisorCheckResultResult): TrustedAdvisorDetailsResult[A] = {
-    TrustedAdvisorDetailsResult(
+  def parseTrustedAdvisorCheckResult[A <: TrustedAdvisorCheckDetails](parseDetails: TrustedAdvisorResourceDetail => Attempt[A], ec: ExecutionContext)(result: DescribeTrustedAdvisorCheckResultResult): Attempt[TrustedAdvisorDetailsResult[A]] = {
+    implicit val executionContext: ExecutionContext = ec
```

Comment:
As discussed IRL this isn't ideal, probably a sign that we should simplify the approach slightly in the future.

Reply:
(but fine for now)